### PR TITLE
Pandaria - History

### PR DESCRIPTION
**Pull request by ghost**
## Developer changelog:
- Generated provinces, landed titles, and holdings for Pandaria.
- Set up object map positions for Pandaria.
- Assigned countless new characters.
- Additional load screens.

## TODO:
- [x] Generate dynasties and characters
- [x] Generate/create historical setups for title holders
- [x] Update _z_landed_titles_pandaria.txt_ to correspond with the _ProvinceDef.xls_ document's color codes, change duchy/kingdom/empire capitals if/wherever necessary for lore/logical reasons
- [x] Create titular kingdoms/duchies and holy orders for major Pandarian factions such as Shado-Pan
- [x] Localization

# How to test:
Start the game and take a look at characters, provinces, titles. Lore knowledge and feedback are appreciated. There are no Pandaria mechanics, just history.

# Notes:
- ~~There may be characters that are displayed as holding a certain title, but chronologically are dead due to a character generator bug.~~
- Some lore characters might be misssing their proper employers.
- ~~Some generated characters might have blank names (they will appear as 'dynasty' in title history)~~
- ~~Some really old title holders (such as Empress Shek'zeer) might die immediately after unpausing. This is caused by lack of longevity mechanics being assigned to the race trait at the moment.~~ Solved.
- Government types and succession mechanics are not final and are subject to change in the mechanics branch.

### CODE REVIEW

File: common/bloodlines/wc_bloodlines.txt [hunk 1]
```diff
@@ -389,4 +389,18 @@ ymiron = {
 	allow_bastards = yes
```
Comment (zumbak04) [inline common/bloodlines/wc_bloodlines.txt:389]: Over demesne limit:
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Looks like many lore characters aren't linked to the gamepedia. Like Shek'zeer.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Errors:
<details>
<summary>Click to expand</summary>

[characterhistory.cpp:545]: Bad Father for character: Yi-Mo (500259)
[characterhistory.cpp:545]: Bad Father for character: Kao (520669)
[province.cpp:5021]: Too many settlements added to province Klaxxi'vess
[province.cpp:5021]: Too many settlements added to province Stormstout
[region.cpp:149]: Region 'island_pandaria' have multiple entries for the province '1240'
[region.cpp:149]: Region 'island_pandaria' have multiple entries for the province '1241'
[region.cpp:149]: Region 'island_pandaria' have multiple entries for the province '1242'
[region.cpp:149]: Region 'world_pandaria_jade_forest' have multiple entries for the province '1240'
[region.cpp:149]: Region 'world_pandaria_jade_forest' have multiple entries for the province '1241'
[region.cpp:149]: Region 'world_pandaria_jade_forest' have multiple entries for the province '1242'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1292'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1233'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1234'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1235'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1236'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1237'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1239'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1238'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1229'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1231'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1232'
[region.cpp:180]: Region 'world_pandaria' have multiple entries for the province '1230'
[title.cpp:12180]: Hoku of c_kardakahar(521111) holds title c_kardakahar while scripted as DEAD in 583.1.1

</details>

**Comment by zumbak04** [comment]
More errors:
<details>
<summary>Click to expand</summary>

```
[persistent.cpp:40]: Error: "Undefined modifier type! token: waterspeaker_opinion, near line: 114" in file: "common/religions/elemental_lords_group.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: neptulon_worship_opinion, near line: 106" in file: "common/religions/pandaren_religion_group.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: hozen_opinion, near line: 1969" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: jinyu_opinion, near line: 1970" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: grummle_opinion, near line: 1971" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: jinyu_group_opinion, near line: 2021" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: pandaren_opinion, near line: 2022" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: grummle_opinion, near line: 2023" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: hozen_opinion, near line: 2024" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: jinyu_group_opinion, near line: 2039" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: pandaren_opinion, near line: 2040" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: grummle_opinion, near line: 2041" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: hozen_opinion, near line: 2042" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: jinyu_group_opinion, near line: 2103" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: pandaren_opinion, near line: 2104" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: hozen_opinion, near line: 2105" in file: "common/traits/wc_race_traits.txt"
[persistent.cpp:40]: Error: "Undefined modifier type! token: jinyu_group_opinion, near line: 2126" in file: "common/traits/wc_race_traits.txt"
```

</details>

**Comment by zumbak04** [comment]
Ping isn't connected to the dynasty. Mia should be a part of the dynasty, but she's not.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Errors:
<details>
<summary>Click to expand</summary>

```
[history.cpp:645]: Duplicate Historical Character! ID:500370
[history.cpp:645]: Duplicate Historical Character! ID:544
[characterhistory.cpp:519]: Unknown History Command ==>'nick'<==  for Character history/characters/520000_mogu.txt at Line 12
[parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/characters/520000_mogu.txt line: 10244"
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "yes", location: " file: history/characters/520000_mogu.txt line: 10244"
[characterhistory.cpp:519]: Unknown History Command ==>'='<==  for Character history/characters/520000_mogu.txt at Line 10244
[parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/characters/560000_grummle.txt line: 563"
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "yes", location: " file: history/characters/560000_grummle.txt line: 563"
[characterhistory.cpp:519]: Unknown History Command ==>'='<==  for Character history/characters/560000_grummle.txt at Line 563
[parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/characters/560000_grummle.txt line: 917"
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "yes", location: " file: history/characters/560000_grummle.txt line: 917"
[characterhistory.cpp:519]: Unknown History Command ==>'='<==  for Character history/characters/560000_grummle.txt at Line 917
[parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/characters/560000_grummle.txt line: 1563"
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "id", location: " file: history/characters/560000_grummle.txt line: 1563"
[characterhistory.cpp:519]: Unknown History Command ==>'='<==  for Character history/characters/560000_grummle.txt at Line 1563
[parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/characters/560000_grummle.txt line: 1819"
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "yes", location: " file: history/characters/560000_grummle.txt line: 1819"
[characterhistory.cpp:519]: Unknown History Command ==>'='<==  for Character history/characters/560000_grummle.txt at Line 1819
```

</details>

**Comment by zumbak04** [comment]
More errors:
<details>
<summary>Click to expand</summary>

```
[titlehistory.cpp:161]: Invalid character 500544 in history/titles/b_honeydew_house.txt
[titlehistory.cpp:167]: Unborn title holder
	Title: c_boulderslide(2100735) in history/titles/c_boulderslide.txt
	Date: 151.8.10
	Character ID: 2100735, Birth date: 156.7.20
[titlehistory.cpp:167]: Unborn title holder
	Title: c_crystal_vice(180279) in history/titles/c_crystal_vice.txt
	Date: 575.12.27
	Character ID: 180279, Birth date: 582.9.17
[titlehistory.cpp:167]: Unborn title holder
	Title: c_frostmane(32045) in history/titles/c_frostmane.txt
	Date: 293.2.25
	Character ID: 32045, Birth date: 297.7.26
[titlehistory.cpp:167]: Unborn title holder
	Title: c_frozen_pass(32045) in history/titles/c_frozen_pass.txt
	Date: 293.2.25
	Character ID: 32045, Birth date: 297.7.26
[titlehistory.cpp:161]: Invalid character 500544 in history/titles/c_honeydew.txt
[titlehistory.cpp:167]: Unborn title holder
	Title: c_kadixxat(540000) in history/titles/c_kadixxat.txt
	Date: 1.1.1
	Character ID: 540000, Birth date: 1.1.2
[titlehistory.cpp:167]: Unborn title holder
	Title: c_karanosh(540000) in history/titles/c_karanosh.txt
	Date: 1.1.1
	Character ID: 540000, Birth date: 1.1.2
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "id", location: " file: history/titles/c_leg_zaol.txt line: 4"
[titlehistory.cpp:331]: Unknown History Command ==>')'<==  for Title in history/titles/c_leg_zaol.txt at Line 4
[parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/titles/c_leg_zaol.txt line: 4"
[parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "id", location: " file: history/titles/c_leg_zaol.txt line: 4"
[titlehistory.cpp:331]: Unknown History Command ==>'='<==  for Title in history/titles/c_leg_zaol.txt at Line 4
[titlehistory.cpp:167]: Unborn title holder
	Title: c_malakajin(2100735) in history/titles/c_malakajin.txt
	Date: 151.8.10
	Character ID: 2100735, Birth date: 156.7.20
[titlehistory.cpp:167]: Unborn title holder
	Title: c_mantivess(540000) in history/titles/c_mantivess.txt
	Date: 1.1.1
	Character ID: 540000, Birth date: 1.1.2
[titlehistory.cpp:167]: Unborn title holder
	Title: c_north_coldwater(32045) in history/titles/c_north_coldwater.txt
	Date: 293.2.25
	Character ID: 32045, Birth date: 297.7.26
[titlehistory.cpp:167]: Unborn title holder
	Title: c_northfield(5581) in history/titles/c_northfield.txt
	Date: 449.1.24
	Character ID: 5581, Birth date: 454.3.3
[titlehistory.cpp:167]: Unborn title holder
	Title: c_northshire(5506) in history/titles/c_northshire.txt
	Date: 107.9.10
	Character ID: 5506, Birth date: 107.10.19
[titlehistory.cpp:167]: Unborn title holder
	Title: c_northshire(5529) in history/titles/c_northshire.txt
	Date: 501.6.1
	Character ID: 5529, Birth date: 506.12.11
[titlehistory.cpp:167]: Unborn title holder
	Title: c_redhelm(11300) in history/titles/c_redhelm.txt
	Date: 300.1.1
	Character ID: 11300, Birth date: 300.2.3
[titlehistory.cpp:167]: Unborn title holder
	Title: c_shekzeer(540000) in history/titles/c_shekzeer.txt
	Date: 1.1.1
	Character ID: 540000, Birth date: 1.1.2
[titlehistory.cpp:167]: Unborn title holder
	Title: c_south_coldwater(32045) in history/titles/c_south_coldwater.txt
	Date: 293.2.25
	Character ID: 32045, Birth date: 297.7.26
[titlehistory.cpp:37]: Putting date entry within another date entry ==>'800.1.1'<==  for Title in history/titles/c_swirlwind.txt at Line 4
[titlehistory.cpp:161]: Invalid character 500544 in history/titles/d_honeydew.txt
[titlehistory.cpp:167]: Unborn title holder
	Title: d_mantivess(540000) in history/titles/d_mantivess.txt
	Date: 1.1.1
	Character ID: 540000, Birth date: 1.1.2
[titlehistory.cpp:167]: Unborn title holder
	Title: e_shekmag(540000) in history/titles/e_shekmag.txt
	Date: 1.1.1
	Character ID: 540000, Birth date: 1.1.2
[titlehistory.cpp:167]: Unborn title holder
	Title: k_frostmane(32045) in history/titles/k_frostmane.txt
	Date: 293.2.25
	Character ID: 32045, Birth date: 297.7.26
```

</details>

File: common/bloodlines/wc_bloodlines.txt (hunk 1 above)
Comment (zumbak04) [inline common/bloodlines/wc_bloodlines.txt:389]: Mudclaw dynasty is too old.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Is she supposed to be 56 years old?
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Ok, basically, it's a problem of all the lore characters. They're too old. Rin born the first child in 48.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Shouldn't Jitong Shan'ze be a shaman?
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Shouldn't they have their nicknames?
<details>
<summary>Click to expand</summary>

image
image
image
image

</details>

**Comment by zumbak04** [comment]
Warlord Kao was resurrected by the Zandalari together with Lei Shen, so he should be dead back then.
He's a theocracy.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Shouldn't he has learnig education.
<details>
<summary>Click to expand</summary>

image

</details>

Shouldn't he have martial education and be warrior or shaman?
<details>
<summary>Click to expand</summary>

image

</details>

Shouldn't he have martial education and be a warrior?
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Shouldn't he have martial education?
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by Malicos** [comment]
> 
> 
> More errors:
> Click to expand
> 
> ```
> [titlehistory.cpp:161]: Invalid character 500544 in history/titles/b_honeydew_house.txt
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_boulderslide(2100735) in history/titles/c_boulderslide.txt
> 	Date: 151.8.10
> 	Character ID: 2100735, Birth date: 156.7.20
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_crystal_vice(180279) in history/titles/c_crystal_vice.txt
> 	Date: 575.12.27
> 	Character ID: 180279, Birth date: 582.9.17
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_frostmane(32045) in history/titles/c_frostmane.txt
> 	Date: 293.2.25
> 	Character ID: 32045, Birth date: 297.7.26
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_frozen_pass(32045) in history/titles/c_frozen_pass.txt
> 	Date: 293.2.25
> 	Character ID: 32045, Birth date: 297.7.26
> [titlehistory.cpp:161]: Invalid character 500544 in history/titles/c_honeydew.txt
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_kadixxat(540000) in history/titles/c_kadixxat.txt
> 	Date: 1.1.1
> 	Character ID: 540000, Birth date: 1.1.2
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_karanosh(540000) in history/titles/c_karanosh.txt
> 	Date: 1.1.1
> 	Character ID: 540000, Birth date: 1.1.2
> [parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "id", location: " file: history/titles/c_leg_zaol.txt line: 4"
> [titlehistory.cpp:331]: Unknown History Command ==>')'<==  for Title in history/titles/c_leg_zaol.txt at Line 4
> [parser.cpp:688]: Script Assert! assert: "!_Toklhs.Match(_eq_)", type: "id", location: " file: history/titles/c_leg_zaol.txt line: 4"
> [parser.cpp:697]: Script Assert! assert: "false && "Statement missing equals sign"", type: "id", location: " file: history/titles/c_leg_zaol.txt line: 4"
> [titlehistory.cpp:331]: Unknown History Command ==>'='<==  for Title in history/titles/c_leg_zaol.txt at Line 4
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_malakajin(2100735) in history/titles/c_malakajin.txt
> 	Date: 151.8.10
> 	Character ID: 2100735, Birth date: 156.7.20
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_mantivess(540000) in history/titles/c_mantivess.txt
> 	Date: 1.1.1
> 	Character ID: 540000, Birth date: 1.1.2
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_north_coldwater(32045) in history/titles/c_north_coldwater.txt
> 	Date: 293.2.25
> 	Character ID: 32045, Birth date: 297.7.26
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_northfield(5581) in history/titles/c_northfield.txt
> 	Date: 449.1.24
> 	Character ID: 5581, Birth date: 454.3.3
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_northshire(5506) in history/titles/c_northshire.txt
> 	Date: 107.9.10
> 	Character ID: 5506, Birth date: 107.10.19
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_northshire(5529) in history/titles/c_northshire.txt
> 	Date: 501.6.1
> 	Character ID: 5529, Birth date: 506.12.11
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_redhelm(11300) in history/titles/c_redhelm.txt
> 	Date: 300.1.1
> 	Character ID: 11300, Birth date: 300.2.3
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_shekzeer(540000) in history/titles/c_shekzeer.txt
> 	Date: 1.1.1
> 	Character ID: 540000, Birth date: 1.1.2
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: c_south_coldwater(32045) in history/titles/c_south_coldwater.txt
> 	Date: 293.2.25
> 	Character ID: 32045, Birth date: 297.7.26
> [titlehistory.cpp:37]: Putting date entry within another date entry ==>'800.1.1'<==  for Title in history/titles/c_swirlwind.txt at Line 4
> [titlehistory.cpp:161]: Invalid character 500544 in history/titles/d_honeydew.txt
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: d_mantivess(540000) in history/titles/d_mantivess.txt
> 	Date: 1.1.1
> 	Character ID: 540000, Birth date: 1.1.2
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: e_shekmag(540000) in history/titles/e_shekmag.txt
> 	Date: 1.1.1
> 	Character ID: 540000, Birth date: 1.1.2
> [titlehistory.cpp:167]: Unborn title holder
> 	Title: k_frostmane(32045) in history/titles/k_frostmane.txt
> 	Date: 293.2.25
> 	Character ID: 32045, Birth date: 297.7.26
> ```

Fixed only the problems related directly to the branch, the rest is subject to Bug Crusade PR.

**Comment by zumbak04** [comment]
Links of Marlfur dynasty are broken, they lead to wrong characters.
Tong, father of Zhang Malfur, isn't Marlfur.
Mia, daughter of Zhang Malfur, isn't Marlfur.

**Comment by zumbak04** [comment]
Links of Blackbrow dynasty are broken, they lead to wrong characters. Lang's link leads to Rin Blackbrow.

**Comment by zumbak04** [comment]
<details>
<summary>Errors</summary>

```
[title.cpp:12180]: Ji of c_hundred_steps(500728) holds title c_hundred_steps while scripted as DEAD in 605.6.6
[title.cpp:12180]: Ji of c_hundred_steps(500728) holds title c_krosh while scripted as DEAD in 605.6.6
```

</details>

**Comment by zumbak04** [comment]
A 13-year-old girl is married.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by BronzeBaron** [comment]
Goldlen Lotus is very weak in comparison its neighbors. 
 - Mogu neighbour starts with 5k~ troops where the Golden Lotus has close to 650~ 
 - Within a few years, Mountain Hozen and Mogu that border had taken their lands without much effort
 - Other Holy Orders are much more viable in their respective regions in comparison, and aren't as easily overtaken
 - Even with a couple decades of progress, I have not seen Golden Lotus catch up to it's neighbors
 - Perhaps balkanize Xin'ze Kingdom a little bit in the first bookmark, so that the Kingdom is not yet founded and infighting can give Golden Lotus a better chance

I found that Yaungol nations in Townlong are very easy pickings for the Mantid, and never seem to last very long
 - Yaungol in the south has enough nomadic retinue to stand up against the Mantid, Townlong Yaungol does not
 - Maybe a mercenary company cheaper than the already offered Zandalari company from the start to open the path for some low quality meat shield soldiers early on

From what I understand, Xin the Weaponmaster had three clans under him (Ming the Cunning, from Clan Harthak, Kuai the Brute from Clan Gurthan, and Haiyan the Unstoppable from Clan Kargesh) by the time he is encountered by adventurers in WoW; Perhaps having these clans independent in the first bookmark would make the region more interesting. Given that Xin'Ze starts with more than enough counties, giving some of these to mentioned clans could offset the imbalance of power while pulling closer to the lore. Will play more, great job all around so far with this.

**Comment by Malicos** [comment]
> Goldlen Lotus is very weak in comparison its neighbors.
> 
>     * Mogu neighbour starts with 5k~ troops where the Golden Lotus has close to 650~
> 
>     * Within a few years, Mountain Hozen and Mogu that border had taken their lands without much effort
> 
>     * Other Holy Orders are much more viable in their respective regions in comparison, and aren't as easily overtaken
> 
>     * Even with a couple decades of progress, I have not seen Golden Lotus catch up to it's neighbors
> 
>     * Perhaps balkanize Xin'ze Kingdom a little bit in the first bookmark, so that the Kingdom is not yet founded and infighting can give Golden Lotus a better chance

Will consider splitting off Shao-Tien off the main Mogu kingdom in 583 bookmark to give Golden Lotus a much more fair chance of expanding in the Vale.

> From what I understand, Xin the Weaponmaster is described as a new King had three subjugated clans under him (https://wow.gamepedia.com/A_New_Lesson_for_the_Master - Ming the Cunning from Clan Harthak, Kuai the Brute from Clan Gurthan, and Haiyan the Unstoppable from Clan Kargesh https://wow.gamepedia.com/Trial_of_the_King ) by the time he is encountered by adventurers in WoW; Perhaps having these clans independent in the first bookmark would make the region more interesting. Given that Xin'Ze starts with more than enough counties, giving some of these to mentioned clans could offset the imbalance of power while pulling closer to the lore.

Generally, these clans are scattered around Pandaria in all bookmarks and do not serve as direct vassals, as it was hard to determine their actual presence beyond Mogu'shan, so I picked several locations that had Mogu presence without a major affiliation to fill the gaps. They might serve as tributary states rather than direct vassals due to distance from the kingdom itself.

**Comment by zumbak04** [comment]
> * Mogu can’t raid infidel neighbors for loot. Is this intended?

Yes. Holy Light mogu should be nice and friendly.

> * Dynasty coats of arms do not always correspond to a character’s race like mogu having “monkey” CoA and hozen having “fish” CoA;

It can't be corresponded unless we remove all the fish and monkey CoAs.

> * Two baronies named the same but in different provinces: Mogu’shan barony in Mogu’shan and Maker’s Perch provinces. Maybe you will want to rename one of them, not sure if it’s relevant;

On it.

> Provinces that lack tech advances:

On it.

**Comment by zumbak04** [comment]
> * House Stormstout (Stormstout province – 3rd bookmark).

Confirmed.

These aren't connected to the dynasty.
<details>
<summary>Click to expand</summary>

image

</details>

Many characters aren't a part of the dynasty.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by Malicos** [comment]
Split the Stormstouts into two dynasties; one for Mainland and others for Wandering Isle relatives of Chen.
Also the characters in Wandering Isle branch are generally purged, aside from those that leave the turtle in a certain time period.

**Comment by zumbak04** [comment]
Still broken.
<details>
<summary>Click to expand</summary>

image
image
image

</details>

**Comment by Malicos** [comment]
Alright, confirmed that purged characters and their descendants lose their assigned dynasty, but whenever they are spawn as active characters, they retain their dynasty.
ID 500179 (Yun), alongside his descendants, 500181 (Chon-po) and 500182 (Chen) in 583 start.
ID 500181 (Chon-po) alongside his descendants, 500183 (Shisai) and 500184 (Li-Li) in 603/605 start.
Not sure what exactly causes this problem.

**Comment by zumbak04** [comment]
Can't say I like 2 different dynasties named the same.
<details>
<summary>Click to expand</summary>

image

</details>

**Pull request merged**